sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessErrorEventArgs.cs

```diff
@@ -17,15 +17,15 @@ public sealed class ProcessErrorEventArgs : EventArgs
         ///
         /// <param name="exception">The exception that this event data belongs to.</param>
         /// <param name="action">The action associated with the message.</param>
-        /// <param name="endpoint">The endpoint used when this exception occurred.</param>
-        /// <param name="entityName">The entity path used when this exception occurred.</param>
+        /// <param name="fullyQualifiedNamespace">The endpoint used when this exception occurred.</param>
```

Comment:
Can we update the comment and also add this `This is likely to be similar to <c>{yournamespace}.servicebus.windows.net</c>`?